intra-factory_transport/requirements.txt

```diff
@@ -0,0 +1,16 @@
+# Core dependencies
+numpy>=1.20.0
+pandas>=1.3.0
+matplotlib>=3.4.0
+scipy>=1.7.0
+
+# Jupyter environment
+jupyter>=1.0.0
+notebook>=6.4.0
+ipykernel>=6.0.0
+
+# Visualization
+plotly>=5.3.0
+
+# NVIDIA cuOpt SDK
+nvidia-cuopt>=1.0.0
```

Comment:
Lets move cuopt installation to main so we can update cuOpt version in one place.